App crashes when share tracks fab pressed

This is the stacktrace. This happens when we open a bookmarked tracked and then go back to the tracks list and share the tracks list.
```
06-22 01:53:47.634 25980-25980/org.fossasia.openevent E/AndroidRuntime: FATAL EXCEPTION: main
                                                                        Process: org.fossasia.openevent, PID: 25980
                                                                        java.lang.RuntimeException: Unable to instantiate service org.fossasia.openevent.Services.BookmarkAlarmService: java.lang.InstantiationException: java.lang.Class<org.fossasia.openevent.Services.BookmarkAlarmService> has no zero argument constructor
                                                                            at android.app.ActivityThread.handleCreateService(ActivityThread.java:2862)
                                                                            at android.app.ActivityThread.-wrap4(ActivityThread.java)
                                                                            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1427)
                                                                            at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                            at android.os.Looper.loop(Looper.java:148)
                                                                            at android.app.ActivityThread.main(ActivityThread.java:5422)
                                                                            at java.lang.reflect.Method.invoke(Native Method)
                                                                            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726)
                                                                            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616)
                                                                         Caused by: java.lang.InstantiationException: java.lang.Class<org.fossasia.openevent.Services.BookmarkAlarmService> has no zero argument constructor
                                                                            at java.lang.Class.newInstance(Native Method)
                                                                            at android.app.ActivityThread.handleCreateService(ActivityThread.java:2859)
                                                                            at android.app.ActivityThread.-wrap4(ActivityThread.java) 
                                                                            at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1427) 
                                                                            at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                            at android.os.Looper.loop(Looper.java:148) 
                                                                            at android.app.ActivityThread.main(ActivityThread.java:5422) 
                                                                            at java.lang.reflect.Method.invoke(Native Method) 
                                                                            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:726) 
                                                                            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:616) 
```

--- a/android/app/src/main/java/org/fossasia/openevent/Services/BookmarkAlarmService.java
+++ b/android/app/src/main/java/org/fossasia/openevent/Services/BookmarkAlarmService.java
@@ -25,6 +25,9 @@ public class BookmarkAlarmService extends IntentService {
         super(name);
     }
 
+    public BookmarkAlarmService(){
+        super("BookMarkAlarmService");
+    }
 
     @Override
     public IBinder onBind(Intent arg0) {

